src/Bridge/Symfony/Validator/Metadata/Property/ValidatorPropertyMetadataFactory.php

```diff
@@ -83,26 +92,23 @@ public function create(string $resourceClass, string $name, array $options = [])
 
         $required = $propertyMetadata->isRequired();
         $iri = $propertyMetadata->getIri();
+        $schema = $propertyMetadata->getSchema();
 
-        if (null !== $required && null !== $iri) {
+        if (isset($required, $iri, $schema)) {
```

Comment:
Why isset? These default values are null

Reply:
`isset` is the same as checking that all values ​​are not null

Reply:
Maybe but it's a function call, so by definition a bit slower

Reply:
@soyuka OK, no problem, I'll change that moment

Reply:
fixed